Fix:
tkrawtcp.c:658:3: error: implicit declaration of function 'TkBindError' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
                TkBindError(interp);
                ^
tkrawtcp.c:665:3: error: implicit declaration of function 'TkBindError' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
                TkBindError(interp);
                ^
tkrawtcp.c:672:3: error: implicit declaration of function 'TkBindError' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
                TkBindError(interp);
                ^
tkrawtcp.c:680:3: error: implicit declaration of function 'TkBindError' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
                TkBindError(interp);
                ^
--- src/tk/tkrawtcp.c.orig	2018-06-29 04:19:31.000000000 -0500
+++ src/tk/tkrawtcp.c	2022-05-16 20:17:24.000000000 -0500
@@ -43,6 +43,7 @@
 #include <sys/un.h>
 
 #include <tk.h>
+#include "tkint.h"
 
 static int inet_connect _ANSI_ARGS_((char *host, char *port,int server));
 static int unix_connect _ANSI_ARGS_((char *path, int server));
